apps/drupal/templates/block/block--system-branding-block.html.twig

```diff
@@ -22,8 +22,8 @@
 
   {# Use logo to explicitly override values from preprocess. #}
   {% set logo = {
-    logo_svg_inline: logo_inline_svg ?: logo_inline_svg,
-    logo_img_src: logo_img_src ?: logo_img_src
+    logo_svg_inline: logo_inline_svg,
```

Comment:
Trying to verify the `logo_svg_inline` vs `logo_inline_svg` naming weirdness here, while lining it up with the var name mentioned in the docblock at top of file.